terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/exceptionhandling/ExceptionHandlingTest.java

```diff
@@ -87,17 +87,19 @@ public void test02_01_useCaseControllerHandling() {
 
         driver.findElement(By.id("useCaseControllerHandling_02_01")).click();
 
+        long retryTimes = 1L;
         assertThat(driver.getTitle(), is("terasoluna-gfw-functionaltest"));
         assertThat(dbLogProvider.countContainsMessageAndLevelsAndLogger(
                 "\\[e.xx.9999\\] 2_1 Continue", "INFO",
                 "org.terasoluna.gfw.common.exception.ExceptionLogger"), is(1L));
         assertThat(dbLogProvider.countContainsMessageAndLevelsAndLogger(
                 "\\[e.xx.9999\\] 2_1 Continue", "INFO",
                 "org.terasoluna.gfw.common.exception.ExceptionLogger.Monitoring"),
-                is(1L));
+                is(retryTimes));
 
         assertThat(dbLogProvider.countContainsByRegexExceptionMessage(null,
-                null, "2_1 Continue*", "..*ContinueException..*"), is(1L));
+                null, "2_1 Continue*", "..*ContinueException..*"), is(
+                        retryTimes));
```

Comment:
- [ ] Unnecessary changes. (HtmlUnit unique.)

Reply:
the fix has been restored.
https://github.com/terasolunaorg/terasoluna-gfw-functionaltest/pull/806/commits/195d8eb7a5d5506d2699add55dd77be1804b02c1